app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1197,6 +1203,7 @@ class BrowserTabFragment :
     ) {
         webViewContainer.gone()
         newBrowserTab.newTabLayout.gone()
+        newBrowserTab.newTabContainerLayout.gone()
```

Comment:
Edge case landing here during onboarding, but this can be removed based on what we saw during testing

Reply:
scratch that, it's correct to set newtab gone here.